x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_form/form_row_setting.tsx

```diff
@@ -125,7 +125,7 @@ export function FormRowSetting({ row, value, onChange, isDisabled }: Props) {
     case 'yaml': {
       return (
         <FixedHeightDiv>
-          <CodeEditor
+          <CodeEditor // TODO Unable to find context that provides theme. Need CODEOWNER Input
```

Comment:
@ogupte Does this not have access to the Kibana context supplied here:
https://github.com/elastic/kibana/blob/8861c27f44e70e8853e508887ee599e8f4a4341e/x-pack/plugins/apm/public/components/routing/app_root/index.tsx#L69

(I'm guessing it might be embedded outside our normal application)

Reply:
@sqren @ogupte Has anyone been able to verify if this editor is working?

Reply:
Last one here - let me know if I can be of assistance to get this verified!

Reply:
This is from a component that is rendered in the Fleet UI (via fleet.registerExtension (https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/public/plugin.ts#L407-L439). It appears to be getting the correct theming with manual testing:
![Screenshot 2023-07-11 at 00 04 40](https://github.com/elastic/kibana/assets/1967266/3e77d4e9-8eaa-4aed-93d7-0f9c8887bce3)

Reply:
Excellent - thank you for verifying!

Reply:
I just tested the user profile only setting, and it appears the CodeEditor picks it up just fine, but the rest of the Fleet UI doesn't appear to use the correct theme:
![Screenshot 2023-07-11 at 09 45 28](https://github.com/elastic/kibana/assets/1967266/9d89a2bc-5d1e-4f3e-b48a-415ad1854736)

Reply:
This PR is only to address CodeEditors.

I created an [issue](https://github.com/elastic/kibana/issues/161135) for the Fleet Dark Mode UI issues. 

Here is the Dark Mode Issues meta [issue](https://github.com/elastic/kibana/issues/158201)